CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Latest Changes
```

Comment:
@std4lqi Thanks for addressing my earlier comments! One more very minor request: Could you also please change "Latest" to "Recent" here so the Jenkins check can pass?

<img width="411" alt="Screen Shot 2021-04-28 at 3 03 21 AM" src="https://user-images.githubusercontent.com/45975633/116361247-c5e2aa80-a7ce-11eb-8c12-3696cb8172cb.png">

Reply:
I see it now, and update it. Thanks! :-)